src/plugins/ssh/init.py

```diff
@@ -49,9 +70,20 @@ def main():
     cmd_params = [jobssh]
 
     if "userssh" in parameters:
-        if "type" in parameters["userssh"] and "value" in parameters["userssh"]:
+        # get user public keys from rest server
+        application_token = plugin_config.get("application_token")
+        username = os.environ.get("PAI_USER_NAME")
+        public_keys = []
+        if application_token:
+            public_keys = get_user_public_keys(application_token, username)
+
+        if "value" in parameters["userssh"]:
```

Comment:
` if "value" in parameters["userssh"] and parameters["userssh"]["value"] != ""`

Reply:
After we modified webportal, `parameters["userssh"]["value"]` could be `""`

Reply:
> After we modified webportal, `parameters["userssh"]["value"]` could be `""`

Fixed, thx